src/aleph/tcp.clj

```diff
@@ -89,12 +89,14 @@
          shutdown-timeout netty/default-shutdown-timeout}
     :as options}]
   (netty/start-server
-   {:pipeline-builder (fn [^ChannelPipeline pipeline]
-                        (.addLast pipeline
-                                  "handler"
-                                  (server-channel-handler handler options))
-                        (pipeline-transform pipeline))
-    :ssl-context ssl-context
+   {:pipeline-builder (cond-> (fn [^ChannelPipeline pipeline]
+                                (.addLast pipeline
+                                          "handler"
+                                          (server-channel-handler handler options))
+                                (pipeline-transform pipeline))
+                        (some? ssl-context)
+                        (netty/add-ssl-handler-to-pipeline-builder
```

Comment:
I got the fact it's just a refactor, but I would have expected to see the `ssl-handler` removed somewhere else on the code base (since it was set already).
Or is it working because we override the handler named `ssl-handler` that was defined elsewhere?

Reply:
`ssl-handler` used to be added by `aleph.netty/start-server` here because we just passed down `ssl-context` to it. Note how we now don't do that anymore, so there's nothing to remove. Does that answer your question? :pray:

Reply:
Sorry for the noise! With your comment, I finally found what I sought [1]. I should have hide whitespaces in the first place.

[1] : https://github.com/clj-commons/aleph/pull/720/files#diff-f5f12719e93b55238daf11207b0aac51587253b0a370237101051d0a30885d4eL97

Reply:
No worries at all, this kind of review much more valuable than just rubber-stamping :+1:  And yeah, hiding whitespace changes ftw :sweat_smile: